src/tests/unit.rs

```diff
@@ -0,0 +1,70 @@
+use crate::tests::comp;
+
+#[test]
+fn hello() {
+    comp(
+        r#"
+        echo "hello"
+        "#,
+        r#"
+        echo "hello"
+        "#,
+    );
+}
+
+#[test]
+fn basic() {
+    comp(
+        r#"
+        import * from "std"
+
+        // Comment
+        let count = 1
+        let age = count > 1
+            then 18
+            else 28
+
+        age += 5
+        let name = "John"
+        let age = 18
+        echo "Hi, I'm {name}. I'm {age} years old."
+        "#,
+        r#"
+        __0_count=1
```

Comment:
okay but this will be a pain in the ass to maintain.

compiled output should be expected to change as features rolled out, as well as there have been talks to simplify emitted variable names, remove `awk` and `bc` dependency and #72

Reply:
@b1ek the point is to have just a few of these tests. This way we know if something does not blow out of the water. But I agree that this is something that is more suited for the stable release

Reply:
so maybe we can do a minimum of tests?
Like for importing and echo?